amida-tech/indaba-client
# Bug/INBA-524 Fix user and project state after creating a new group

#### What does this PR do?
Load project and users after creating a new group to get their latest state in redux

#### Related JIRA tickets:
[INBA-524](https://jira.amida-tech.com/browse/INBA-524)

#### How should this be manually tested?
1. Add a group to a project
1. Check that the group appears in the project state and the users in that group without a refresh

#### Background/Context

#### Screenshots (if appropriate):
